src/boot/z_std_dma.c

```diff
@@ -132,27 +135,29 @@ s32 DmaMgr_DmaRomToRam(uintptr_t rom, void* ram, size_t size) {
         ram = (u8*)ram + buffSize;
     }
 
-    if (1) {} // Also necessary to match
+    if (1) { // Also necessary to match
+        s32 pad[2];
 
```

Comment:
```suggestion
    if (1) { // Also necessary to match
        s32 pad[2];
    }
```
would prefer to not indent the whole thing if we can avoid it

Reply:
I thought that the padding was some optimized-out local variables for the following code, and maybe we could figure out what they were later, but I don't mind the unindent